packages/content-management/favorites/favorites_public/src/components/favorites_empty_state.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiEmptyPrompt, useEuiTheme, EuiImage } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React from 'react';
+import emptyFavoritesDark from './empty_favorites_dark.svg';
+import emptyFavoritesLight from './empty_favorites_light.svg';
+
+export const FavoritesEmptyState = ({
+  emptyStateType = 'noItems',
+  entityNamePlural = i18n.translate('contentManagement.favorites.defaultEntityNamePlural', {
+    defaultMessage: 'items',
+  }),
+  entityName = i18n.translate('contentManagement.favorites.defaultEntityName', {
+    defaultMessage: 'item',
+  }),
+}: {
+  emptyStateType: 'noItems' | 'noMatchingItems';
+  entityNamePlural?: string;
+  entityName?: string;
+}) => {
+  const title =
+    emptyStateType === 'noItems' ? (
+      <FormattedMessage
+        id="contentManagement.favorites.noFavoritesMessageHeading"
+        defaultMessage="You haven’t starred any {entityNamePlural}"
+        values={{ entityNamePlural }}
+      />
+    ) : (
+      <FormattedMessage
+        id="contentManagement.favorites.noMatchingFavoritesMessageHeading"
+        defaultMessage="No starred {entityNamePlural} match your search"
+        values={{ entityNamePlural }}
+      />
+    );
+
+  return (
+    <EuiEmptyPrompt
+      css={css`
+        .euiEmptyPrompt__icon {
+          min-inline-size: 25%; /* reduce the min size of the container to fit more title in a single line* /
+        }
+      `}
+      layout="horizontal"
+      color="transparent"
+      icon={<NoFavoritesIllustration />}
+      hasBorder={false}
+      title={<h2>{title}</h2>}
+      body={
+        <FormattedMessage
+          id="contentManagement.favorites.noFavoritesMessageBody"
+          defaultMessage="You can easily star {entityNamePlural} to keep them handy. Simply click the star icon next to a {entityName}, and it will be added to your Starred list."
```

Comment:
```suggestion
          defaultMessage="Keep track of your most important {entityNamePlural} by adding them to your **Starred** list. Click the {starIcon} **star icon** next to a dashboard name and it'll appear in this tab."
```
@andreadelrio 
I usually try to avoid adverbs like "simply" or "easily" (though it is simple and easy in this case) as we tend to overuse them in this type of messaging. It's not a best practice for UI copy in general.

It'd be great to show the icon like this in the body, if that is feasible? 
<img width="144" alt="image" src="https://github.com/user-attachments/assets/676f0422-8b71-4110-b90a-d9984bdd6a94">

Feel free to adapt or ignore the suggestion if it derails too much from the original intent.

Reply:
Thank you, with the markdown for bold emphasis, I could add an icon using unicode symbol ✩, not eui icon. 
Can this work? 

![Screenshot 2024-08-05 at 12 58 30](https://github.com/user-attachments/assets/be7aa6a8-3d96-47d2-bf45-1aee8c4f891e)

Reply:
Works for me 👍 Thanks!